python/cudf/cudf/core/dataframe.py

```diff
@@ -6528,9 +6531,10 @@ def to_struct(self, name=None):
         name: optional
             Name of the resulting Series
 
-        Notes
-        -----
-        Note that a copy of the columns is made.
+        .. pandas-compat::
+            **DataFrame.to_struct**
+
+            Note: that a copy of the columns is made.
```

Comment:
I think that pandas dataframes don't support a `to_struct` method (they don't have struct dtypes because its just an object), so this is (I think) not a pandas compatibility note.